apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -152,4 +154,38 @@ defmodule Lexical.RemoteControl do
   defp asdf?, do: is_binary(System.find_executable("asdf"))
 
   defp rtx?, do: is_binary(System.find_executable("rtx"))
+
+  # We launch lexical by asking the version managers to provide an environment,
+  # which contains path munging. This initial environment is present in the running
+  # VM, and needs to be undone so we can find the correct elixir executable in the project.
+  defp reset_env(:asdf, _root_path) do
+    orig_path = System.get_env("PATH_SAVE", System.get_env("PATH"))
+
+    Enum.map(System.get_env(), fn
+      {"ASDF_ELIXIR_VERSION", _} -> {"ASDF_ELIXIR_VERSION", nil}
+      {"ASDF_ERLANG_VERSION", _} -> {"ASDF_ERLANG_VERSION", nil}
+      {"PATH", _} -> {"PATH", orig_path}
+      other -> other
+    end)
+  end
+
+  defp reset_env(:rtx, root_path) do
+    {env, _} = System.cmd("rtx", ~w(), cd: root_path)
+
+    env
+    |> String.trim()
+    |> String.split("\n")
+    |> Enum.map(fn "export " <> key_and_value ->
+      [key, value] =
+        key_and_value
+        |> String.split("=")
```

Comment:
```suggestion
        |> String.split("=", parts: 2)
```
This would avoid breaking the pattern match if the value itself contains "="

Reply:
excellent catch.